LICENSE-CODE

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
The official license text uses http, and I'd like to match the official text, to the letter.
https://www.apache.org/licenses/LICENSE-2.0

I'd suggest undoing the changes to this file.

Reply:
Done